pkg/command/root.go

```diff
@@ -291,13 +331,22 @@ func newRootCmd() *cobra.Command {
 }
 
 // setLoggerVerbosity sets the verbosity of the logger if TANZU_CLI_LOG_LEVEL is set
-func setLoggerVerbosity() {
-	// Configure the log level if env variable TANZU_CLI_LOG_LEVEL is set
-	logLevel := os.Getenv(log.EnvTanzuCLILogLevel)
-	if logLevel != "" {
-		logValue, err := strconv.ParseInt(logLevel, 10, 32)
-		if err == nil {
-			log.SetVerbosity(int32(logValue))
+func setLoggerVerbosity(verbosity int) {
+	// If verbose global flag is passed set the log level verbosity  if not then check if env variable TANZU_CLI_LOG_LEVEL is set
+	if verbosity > 0 {
+		// Set the log level verbosity with the verbosity value
+		log.SetVerbosity(int32(verbosity))
+
+		// Set the TANZU_CLI_LOG_LEVEL env with the verbosity value
+		_ = os.Setenv(log.EnvTanzuCLILogLevel, strconv.Itoa(verbosity))
```

Comment:
For plugins to consume transparently. Nice!  

In the unit test you write, can you check that this variable is set as expected?